Fixes issue where panel body would always be the height of the screen

josh-burton opened the pull request:
`MediaQuery.of(context).size.height` always returns the full screen height, rather than the height of the widgets parent.

If there is an appbar in a scaffold for example, the panel’s body height should only be as big as its parent, not the height of the screen.

By using Positioned.fill, the body will now fill the stack and be sized correctly.

hectorAguero commented:
So, I change this code, and they fix all problems with a widget that I put in `SlidingPanel(body:` above a BottomBar, previosly I solve with use of padding, but iOS(& Android Q) devices with gesture navigation, have a different bottombarbar height.

The problem is that parallax effect, doesn't work properly... I think that Positioned.fill is changing the body size with every panel sliding coordinate.

Also, `SlidingPanel(panel:` keep taking the entire screen size

cream-cheeze commented:
@josh-burton there is a conflict here.

Maybe you fix it to speed up the PR approve?

josh-burton committed:
Fixes issue where panel body would always be the height of the screen

`MediaQuery.of(context).size.height` always returns the full screen height, rather than the height of the widgets parent.

If there is an appbar in a scaffold for example, the panel’s body height should only be as big as its parent, not the height of the screen.

josh-burton commented:
I've rebased and reapplied the fix. It has been 2 years so I'm not holding out hope this will be merged though.

sladomic commented:
@akshathjain What is the reason of not merging this? What has to be changed in the PR for it to get merged?

AngeloAvv commented:
Any updates? Does a workaround exists?

shahmirzali49 commented:
> So, I change this code in my app libraries for testing, and they fix all size problems with a widget in `SlidingPanel(body:` that I want to put above a BottomBar, previously I solve with use of padding, but iOS(& Android Q) devices with gesture navigation have a different bottombar height, so the padding doesn't work well on that devices.
> 
> The problem is that `SlidingPanel(parallaxEnabled:true` doesn't work properly... I think that Positioned.fill is changing the body size with every panel sliding coordinate.
> 
> Also, `SlidingPanel(panel:` keep taking the entire screen size

did you solve this parallax problem ? @hectorAguero
